src/lib/containers/synapse_table_functions/GetUniqueEntities.ts

```diff
@@ -0,0 +1,22 @@
+import { QueryResultBundle } from '../../utils/synapseTypes/'
```

Comment:
I think this file should be renamed to be camel case - getUniqueEntities, same with the others - getColumnIndiciesWithType, and renderTableCell